src/main/scala/nl/knaw/dans/easy/fedora2vault/EasyFedora2vaultApp.scala

```diff
@@ -89,7 +91,7 @@ class EasyFedora2vaultApp(configuration: Configuration) extends DebugEnhancedLog
       _ <- getManifest(foXml)
         .map(compareManifest(bag))
         .getOrElse(Success(())) // TODO check with sha's from fedora
-    } yield "???" // TODO what?
+    } yield "Created " + msg
```

Comment:
in the end, i would like a csv report containing these fields:
```
easy-dataset-id  input easy-dataset-id
doi              doi as it appears in the EMD
depositor        EASY-User-Account of the depositor of the dataset
transformation   transformation used
UUID             UUID created for the resulting AIP
comments         if the dataset does not conform to the transformation-requirements, it is chronicled here
```

Reply:
I'd rather fix that with #3

Reply:
Do these UUID's imply that the outputDir should not become the root of the created bag(s) but always contain the bags, even when just a single bag is specified?